### ISSUE TITLE
"My Journal" Allows Selecting All Emotions for a Single Day

### ISSUE THREAD

**Issue by vinaybadgujar102** [tijlleenders/ZinZen#1244]
**Describe the bug**
Currently in "My Journal" section, user can add all the emotions in single day.

**To Reproduce**
1. Click on 'Journal' button at bottom navigation
2. Click "+" button and add all emotions.
3. Notice that all emotions are recorded for the day.

**Expected behavior**
The expected behavior is that a person should be able to select one or more emotions from the available options, but not all of them for a single day. Each day should reflect the actual emotions experienced by the user.

**Screenshots**
![ss](https://github.com/tijlleenders/ZinZen/assets/90821253/e5b79861-5346-4126-b1df-56f05e003cfe)

**Additional context**
Users should be able to express their genuine emotions for a given day, but the current behavior allows for the selection of all emotions, which may not accurately represent their emotional state. It would be helpful to limit the selection to a reasonable number of emotions per day or provide a validation check to prevent selecting all emotions.

**Comment by tijlleenders** [comment]
@vinaybadgujar102 
Thanks for the elaborate bug report. I agree your screenshot looks confusing...  
However, the option to add multiple feelings over the day is not a bug but a feature in my opinion. If you feel bad in the morning but good in the afternoon - why not reflect that? The solution might lie in a better visualisation of when the feeling was registered?

**Comment by vinaybadgujar102** [comment]
@tijlleenders I agree with your solution. By enhancing the visual representation, users will have a clearer understanding of the timeline and sequence of their emotions. This could involve incorporating timestamps or a more intuitive layout that clearly shows when each feeling was recorded. Not only would this make the data more visually appealing, but it would also make it easier for users to interpret and analyze their emotional patterns.
